tbx/project_styleguide/templates/patterns/molecules/partner_logos/partner_logos.html

```diff
@@ -1,7 +1,9 @@
 {% load wagtailimages_tags %}
 
-{% for partner_logo in partner_logos %}
-    <div class="partners__logo-wrapper">
-        {% image partner_logo max-100x90 format-webp loading="lazy" class="partners__logo" %}
-    </div>
-{% endfor %}
+<div class="partners__logos {{ classes }}">
```

Comment:
I've always wrapped `{{ classes }}` in a conditional but now i'm questioning if I needed to do that.

Reply:
it's a good point. If the `classes` var isn't present django will just ignore it rather than fail - but I've added in the if statement to make it clearer that it is an optional variable.